src/cli.rs

```diff
@@ -530,6 +530,10 @@ pub struct Opt {
     #[structopt(long = "wrap-right-prefix-symbol", default_value = "…")]
     pub wrap_right_prefix_symbol: String,
 
+    #[structopt(long = "navigate-regex", default_value = "")]
+    /// A regexp to use in the less pager when navigating (auto-generated when empty)
+    pub navigate_regex: String,
```

Comment:
Hi @WayneD, this PR seems spot-on to me. Thanks very much. The only change I suggest is that I feel that it's more correct to use `Option<String>` here with no `default_value`. That avoids giving the impression that `""` is a real default value that's going to be used (and is arguably more idiomatic Rust, letting `None` play the role of a null value).

With that change it will appear as `Option<String>` in the `cli::Opt` struct, but will be converted to `String` in the `config::Config` struct.

Reply:
I finally got around to making this change.  It indeed seems nicer.  The code does treat both None and an empty string as being unspecified since that allows someone to override a default value in their git config with the auto-gen value by specifying an empty string to --navigate-regex.